js/views/modals/Settings/Page.js

```diff
@@ -30,9 +30,33 @@ export default class extends baseVw {
       'click .js-avatarRight': 'avatarRightClick',
       'click .js-headerLeft': 'headerLeftClick',
       'click .js-headerRight': 'headerRightClick',
+      'change .js-colorPicker': 'handleColorChosen',
+      'change .js-colorCode': 'handleColorCodeEntered',
     };
   }
 
+  /** Handles when a hex color code is entered by updating color picker. */
+  handleColorCodeEntered(event) {
+    const colorPickerId = this.$(event.target).data('color-picker-id');
+    const $colorPicker = this.getCachedElement(colorPickerId);
+    const newHexColorCode = event.target.value;
+
+    // If the text passes a basic RegExp for a valid 6 digit hex value,
+    // update the color picker's color.
+    if (/^#([0-9a-f]{6})$/i.test(newHexColorCode)) {
+      $colorPicker.val(newHexColorCode);
+    }
+  }
+
+  /** Handles when a color is chosen from the color picker by updating hex color code text. */
+  handleColorChosen(event) {
+    const hexInputId = this.$(event.target).data('hex-input-id');
```

Comment:
You don't need to change this at all, but as an FYI, in the case when you already have the HTML element, it's not as critical to use `this.$` since it's not going to the DOM anyway... but as long as still works, it's OK.